sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/FilterConjunctionGroup.cs

```diff
@@ -81,7 +81,7 @@ private void CreateFilters(out CollectionConfigurationError[] errors)
                             string.Format(CultureInfo.InvariantCulture, "Failed to create a filter {0}.", filterInfo),
                             e,
                             Tuple.Create("FilterFieldName", filterInfo.FieldName),
-                            Tuple.Create("FilterPredicate", filterInfo.Predicate.ToString()),
+                            Tuple.Create("FilterPredicate", filterInfo.Predicate?.ToString() ?? "null"),
```

Comment:
Predicate is nullable, so need the null check here. This block is creating an error message so I think it's appropriate to use `"null"` here.